src/index.js

```diff
@@ -1,16 +1,18 @@
+const linguistData = require('linguist-languages/data/Solidity.json');
 const { handleComments, printComment } = require('./comments');
-
 const massageAstNode = require('./clean');
 const loc = require('./loc');
 const options = require('./options');
 const parse = require('./parser');
 const print = require('./printer');
 
 // https://prettier.io/docs/en/plugins.html#languages
+const { languageId, ...rest } = linguistData;
 const languages = [
   {
-    extensions: ['.sol'],
-    name: 'Solidity',
+    linguistLanguageId: languageId,
+    ...rest,
```

Comment:
I would like to keep it explicit or else add what's in the `linguistData` object if possible

Reply:
can do the object is very small.